app/src/main/java/com/example/poste/PosteApplication.java

```diff
@@ -49,7 +42,7 @@ public static com.example.poste.api.poste.models.User getCurrentUser() {
      *
      * @param currentUser The User object representing the currently logged-in user
      */
-    public static void setCurrentUser(com.example.poste.api.poste.models.User currentUser) {
+    public static void setCurrentUser(User currentUser) {
```

Comment:
This can (and should) go altogether. Since we are using Singleton, we want to avoid storing a user anywhere as much as possible (in case user logs out or whatnot) and instead just use User.getUser()